config/settings.yml

```diff
@@ -1784,4 +1795,4 @@ schema_contract:
   test_index: 'spec/fixtures/schema_contract/test_schema.json'
 
 form_mock_ae_design_patterns:
-  prefill: true
\ No newline at end of file
+  prefill: true
```

Comment:
Was this meant to be added?

Reply:
a newline at the end of file was automatically added by the IDE, guess for no other reason than a historical standard: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729725#729725.